supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -379,6 +379,28 @@ NOTE: When referring generally to other Kubernetes components, such as pods, nod
 
 *See also*:
 
+[discrete]
+[[operator-hub]]
+==== image:images/yes.png[yes] OperatorHub (noun)
+*Description*: In red Hat OpenShift, the "OperatorHub" is a central location where you can find a wide array of useful Operators built by the community.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[operator-lifecycle-manager]]
+==== image:images/yes.png[yes] Operator Lifecycle Manager (OLM) (noun)
+*Description*: In Red Hat OpenShift, the "Operator Lifecycle Manager" (OLM) helps users to install, update, and manage the lifecycle of Kubernetes native applications (Operators) and their associated services running across their OpenShift Container Platform clusters. It is part of the Operator Framework, an open source toolkit designed to manage Operators in an effective, automated, and scalable way.
```

Comment:
Italics and remove "the"

```suggestion
*Description*: In Red Hat OpenShift, _Operator Lifecycle Manager_ (OLM) helps users to install, update, and manage the lifecycle of Kubernetes native applications (Operators) and their associated services running across their OpenShift Container Platform clusters. It is part of the Operator Framework, which is an open source toolkit designed to manage Operators in an effective, automated, and scalable way.
```
Maybe consider an entry for "Operator Framework" instead of defining it here?